Sources/ComposableArchitecture/Store.swift

```diff
@@ -485,6 +485,9 @@ public final class Store<State, Action> {
     for id in self.children.keys {
       self.invalidateChild(id: id)
     }
+    self.effectCancellables.values.forEach { cancellable in
+      cancellable.cancel()
+    }
```

Comment:
Could this even be:

```suggestion
    self.effectCancellables = [:]
```

?